### DISCUSSION TITLE
Core version in the application ?

### DISCUSSION THREAD

**Discussion by Ing-Dom** [earlephilhower/arduino-pico#480] [Q&A] [answered]
Hi,

is there information about the core version availible in the application?

for things like

```
#if PICO_CORE_V > 1100
Serial.SetPoolingMode()
#endif
```

if not, could be introduced wit build.extra_flags=in boards.txt

**Comment by earlephilhower** [comment, marked as answer]
See https://github.com/earlephilhower/arduino-pico/blob/master/cores/rp2040/RP2040Version.h
